styles/status.cafe/catppuccin.user.css

```diff
@@ -0,0 +1,126 @@
+==/UserStyle== */
+
+@-moz-document regexp("https://status\\.cafe/(?!users).*") {
+    @media (prefers-color-scheme: light) {
+        :root {
+            #catppuccin(@lightFlavor, @accentColor);
+        }
+    }
+    @media (prefers-color-scheme: dark) {
+        :root {
+            #catppuccin(@darkFlavor, @accentColor);
+        }
+    }
+
+    #catppuccin(@lookup, @accent) {
+        @rosewater: @catppuccin[@@lookup][@rosewater];
+        @flamingo: @catppuccin[@@lookup][@flamingo];
+        @pink: @catppuccin[@@lookup][@pink];
+        @mauve: @catppuccin[@@lookup][@mauve];
+        @red: @catppuccin[@@lookup][@red];
+        @maroon: @catppuccin[@@lookup][@maroon];
+        @peach: @catppuccin[@@lookup][@peach];
+        @yellow: @catppuccin[@@lookup][@yellow];
+        @green: @catppuccin[@@lookup][@green];
+        @teal: @catppuccin[@@lookup][@teal];
+        @sky: @catppuccin[@@lookup][@sky];
+        @sapphire: @catppuccin[@@lookup][@sapphire];
+        @blue: @catppuccin[@@lookup][@blue];
+        @lavender: @catppuccin[@@lookup][@lavender];
+        @text: @catppuccin[@@lookup][@text];
+        @subtext1: @catppuccin[@@lookup][@subtext1];
+        @subtext0: @catppuccin[@@lookup][@subtext0];
+        @overlay2: @catppuccin[@@lookup][@overlay2];
+        @overlay1: @catppuccin[@@lookup][@overlay1];
+        @overlay0: @catppuccin[@@lookup][@overlay0];
+        @surface2: @catppuccin[@@lookup][@surface2];
+        @surface1: @catppuccin[@@lookup][@surface1];
+        @surface0: @catppuccin[@@lookup][@surface0];
+        @base: @catppuccin[@@lookup][@base];
+        @mantle: @catppuccin[@@lookup][@mantle];
+        @crust: @catppuccin[@@lookup][@crust];
+        @accent-color: @catppuccin[@@lookup][@@accent];
+
+        color-scheme: if(@lookup = latte, light, dark);
+
+        ::selection {
+            background-color: fade(@accent-color, 30%);
+        }
+
+        input,
+            textarea {
+                &::placeholder {
+                color: @subtext0 !important;
+            }
+        }
+
+        background-color: @base;
+
+        body {
+            background-color: @base;
+            color: @text;
+        }
+        
+        textarea {
+            background-color: @mantle;
+            border-color: @overlay2;
+        }
+        
+        input {
+            background-color: @crust;
+            border-color: @overlay2;
+        }
+        
+        input#homepage, input#email, input#picture {
+                background-color: @mantle !important;
+            }
+
+        .flash {
+            background-color: @green;
+        }
+
+        a:not(.flash a) {
+            color: @accent-color;
+        }
+
+        a:visited:not(.flash a) {
+            color: darken(@accent-color, 5%);
+        }
+    }
+
+    & when (@lookup = latte){
+        .flash {
+            color: darken(@green, 15%);
+            a {
+                color: @blue;
+            }
+        }   
+    }
```

Comment:
I think you meant to put this inside the #catppuccin mixin? Userstyles loads with an error with this file because of this I believe.

Reply:
Fixed! Thanks for pointing it out